virtualizarr/tests/test_integration.py

```diff
@@ -86,7 +97,9 @@ def test_kerchunk_roundtrip_concat(self, tmpdir, format):
         vds.virtualize.to_kerchunk(f"{tmpdir}/refs.{format}", format=format)
 
         # use fsspec to read the dataset from disk via the zarr store
-        roundtrip = xr.open_dataset(f"{tmpdir}/refs.{format}", engine="kerchunk")
+        roundtrip = xr.open_dataset(
+            f"{tmpdir}/refs.{format}", engine="kerchunk", decode_times=decode_times
+        )
 
         # assert equal to original dataset
         xrt.assert_equal(roundtrip, ds)
```

Comment:
This test doesn't pass yet, but it's getting a lot closer! It's basically a floating point difference at this point. I tried switching to `xrt.assert_allclose` but I'm not sure it understands times because it still raises. If I do `(roundtrip.time - ds.time).sum()` I get zero so I do think the times are pretty darn close.

Reply:
I tried using numpy directly to get to the bottom of things:

```python
>>> from numpy.testing import assert_allclose
>>> assert_allclose(roundtrip.time.values, ds.time.values)
*** numpy.exceptions.DTypePromotionError: The DType <class 'numpy.dtypes.DateTime64DType'> could not be promoted by <class 'numpy.dtypes._PyFloatDType'>. This means that no common DType exists for the given inputs. For example they cannot be stored in a single array unless the dtype is `object`. The full list of DTypes is: (<class 'numpy.dtypes.DateTime64DType'>, <class 'numpy.dtypes._PyFloatDType'>)
```

Reply:
See 0ffce4f for a rather silly solution